pkg/model/file/marker.go

```diff
@@ -51,6 +51,8 @@ func (m Marker) String() string {
 	return m.comment + prefix + m.value
 }
 
+// CodeFragments represents a set of code fragments
 type CodeFragments []string
 
+// CodeFragmentsMap binds CodeFragments to their Marker
```

Comment:
Could you please add more info? When and why the CodeFragments is used?
Would be great we doc why we have the  `CodeFragments []string` and `CodeFragmentsMap map[Marker]CodeFragments` as well.